instrumentation/metric/gauge.go

```diff
@@ -48,6 +51,14 @@ func (g *Gauge) Value() int64 {
 	return g.value
 }
 
+func (g *Gauge) Reset() {
```

Comment:
I think it doesn't make sense to reset `Gauge`s at all. They are counters of present state, for instance we have a gauge that counts num of transactions in pool